docs/feature-gates.md

```diff
@@ -250,8 +245,24 @@ metadata:
     ipam.antrea.io/ippools: 'pool1'
 ```
 
+#### Data path change for this feature
+
+When `AntreaIPAM` is enabled, `antrea-agent` will connect the Node's network interface
+to the OVS bridge at startup, and it will detach the interface from the OVS bridge and
+restore its configurations at exit.
+Node may lose network connection when `antrea-agent`or OVS daemons are stopped
```

Comment:
I would suggest not to add new lines for the same paragraph.

Reply:
Fixed.